### DISCUSSION TITLE
Inconsistency of languages in the title of dialog boxes

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#40] [1.1 General Help and Discussion]
Dear Alex, dear community,

Save As... opens a dialog box with in my case a German title (Speichern unter):
<img width="267" alt="Pasted image 20240218145153" src="https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/f1a7251d-7474-4002-be2b-57c29b4c4972">

while Save Copy As... gives the English title:
<img width="128" alt="Pasted image 20240218145215" src="https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/5a50af3d-2b37-4588-bcac-def7a6f971fd">

```GetColor``` also has a German dialog box.

Not that it is a big deal, but in case we want to automate taking screenshots, it might be a point to keep in mind.

WinEdt versions where I tested it: Build: 20230519  (v. 11.1) - 64-bit and Build: 20240319  (v. 11.2) - 64-bit
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
Common Windows Dialogs (Open, Save As, Colors, Print, Font) are localized for your default Windows Language. WinEdt Dialogs are not...
